packages/core/src/geometry/GeometrySystem.ts

```diff
@@ -521,7 +459,7 @@ export class GeometrySystem extends System
         all = Object.keys(this.managedBuffers);
         for (let i = 0; i < all.length; i++)
         {
-            this.disposeBuffer(this.managedBuffers[all[i]], contextLost);
+            this._buffer.dispose(this.managedBuffers[all[i]], contextLost);
```

Comment:
Should this be `this._buffer.managedBuffers` instead of `this.managedBuffers`? These duplicate objects on Buffer and Geometry systems.

Reply:
as managed buffers only really work with geometry, figured I would leave `managedBuffers` there?

Reply:
oh.. actually I see what you mean now 👍